extensions/amp-analytics/0.1/linker-manager.js

```diff
@@ -285,7 +285,7 @@ export class LinkerManager {
     // Don't append linker for exact domain match, relative urls, or
     // fragments.
     const winHostname = WindowInterface.getHostname(this.ampdoc_.win);
-    if (winHostname === hostname) {
+    if (!domains && winHostname === hostname) {
```

Comment:
I still think we should move the check to within the `if(!domains)` check.

Reply:
Done. Also added a bit to docs.